test/liveman/user/users_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule Liveman.User.UsersTest do
+  use Liveman.DataCase, async: true
+
+  alias Liveman.User.Users
+
+  describe "register_user/1" do
+    test "creates a user successfully when the given params are valid" do
+      result = Users.register_user(%{email: "john@doe.com", password: "secret"})
```

Comment:
Instead of result as variable name, it could be `user`:

```suggestion
      user = Users.register_user(%{email: "john@doe.com", password: "secret"})
```